ads/google/utils.js

```diff
@@ -28,13 +28,15 @@ import {user} from '../../src/log';
  * @param {boolean} multiSizeValidation A flag that if set to true will enforce
  *   the rule that ensures multi-size dimensions are no less than 2/3rds of
  *   their primary dimension's counterpart.
+ * @param {boolean=} isFluid Indicates whether this ad slot is Fluid-enabled.
  * @return {?Array<!Array<number>>} An array of dimensions.
  */
 export function getMultiSizeDimensions(
     multiSizeDataStr,
     primaryWidth,
     primaryHeight,
-    multiSizeValidation) {
+    multiSizeValidation,
+    isFluid) {
```

Comment:
Needs a default value.

Reply:
Even if the value defaults to falsey (as desired)?

Reply:
Yes, all optional parameters have to have default values. Otherwise the compiler doesn't know they're optional.

Reply:
Done.